### PR TITLE
Peer review request

### PR DESCRIPTION
**Pull request by dev89s**
Please review my code and check whether I have used JavaScript best practices in my code. 🙏

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todo List</title>
+<script defer src="main.js"></script></head>
```
Comment (vanina123) [inline dist/index.html:9]: Please the script tag should be found at the bottom of the body

**Review by vanina123 — COMMENTED** [review]
Great job so far in your To do list project.
you followed and implemented all the reequipment needed.
HTML CSS and JavaScript best practices✅
DRY, KISS and YAGNI well implemented.✅
so far please try putting your script tag before the closing tag of the body

cheers and happy coding

File: dist/index.html (hunk 1 above)
Comment (dev89s) [reply dist/index.html:9]: Thanks for your comment. 🙏
I believe the file in the dist folder is generated by webpack.

Comment (vanina123) [reply dist/index.html:9]: Hello @dev89s
Yes your right ✅👌,your project is complete
Cheers and happy coding

**Pull request merged**